src/components/AsideBlock/AsideBlock.scss

```diff
@@ -0,0 +1,40 @@
+@import '../../styles/variables';
+
+$asideblock-default-height: calc(100% - 62px);
+$asideblock-default-width: 280px;
+$asideblock-max-width: 70%;
+$asideblock-default-box-shadow: 0 12px 12px 6px rgba(0, 0, 0, 0.08);
```

Comment:
We have box-shadow mixin and variable. Please, reuse the existing one from vars.

Please, study the already existing vars and mixins and common classes that should be reused.